src/StackExchange.Redis/Interfaces/IDatabase.cs

```diff
@@ -2117,7 +2117,7 @@ public interface IDatabase : IRedis, IDatabaseAsync
         /// <param name="flags">The flags to use for this operation.</param>
         /// <returns>The value of key, or nil when key does not exist.</returns>
         /// <remarks>https://redis.io/commands/get</remarks>
-        Lease<byte> StringGetLease(RedisKey key, CommandFlags flags = CommandFlags.None);
+        Lease<byte>? StringGetLease(RedisKey key, CommandFlags flags = CommandFlags.None);
```

Comment:
Ok so the nullable return types from IDatabase are going to basically be those that return a nullable value type, those that return a RedisValue, we're going to continue with the `RedisValue.Null` pattern?

Reply:
Correct - or an array. Given the mismatch today (e.g. if you pass 0 keys to delete, we give an empty array, but if you `.FireAndForget`, we give a null), I normalized on the empty array everywhere I could to reduce breaks and be ore consistent. In the cases like `Least<string>`, it's...just not lying about it, so yeah that's it for reference types. I have some regrets in here like allowing `GeoPosition?` (public API today, which we should have stuck with the `.IsNull` pattern on...but it's shipped).